docs/operators/jsonFormatter.md

```diff
@@ -0,0 +1,25 @@
+Json Formatter
```

Comment:
Please add section for "Operator Usecase", "Operator Information" & "Limitations". You can see http://apex.apache.org/docs/malhar/operators/enricher/ .

Also add any advanced section if required.